SCons/Environment.py

```diff
@@ -1499,10 +1499,10 @@ def _copy2_from_cache(self, src, dst):
     def Decider(self, function):
         copy_function = self._copy2_from_cache
         if function in ('MD5', 'content'):
-            if not md5:
+            if not SCons.Util.md5:
```

Comment:
I made this change to resolve a conflict. Please review and make sure that's correct.

Also probably need to start a deprecation cycle on calling this MD5, along with warnings,etc. 
Can discuss further on discord.